### ISSUE TITLE
Double counting in ETMoses

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#1249]
Currently, we model hot water and space heating separately. This has several unwanted side-effects:
* The investment and O&M costs are payed twice by the stakeholder
* The buffer-size is fully available for both applications
* The capacity of both technologies can be deployed simultaneously

To really solve this in an elegant way, we need to merge space heating and hot water combi-options in the ETM first.

**Comment by ChaelKruip** [comment]
Including @antw @dennisschoenmakers @grdw @jorisberkhout @AlexanderWirtz @DorinevanderVlies

**Comment by grdw** [comment]
> Currently, we model hot water and space heating separately.

Is this for every technology that has a distinction between hot water and space heating? Or are we just talking buffers?

````
buffer_space_heating
households_space_heater_combined_network_gas
households_space_heater_district_heating_steam_hot_water
households_space_heater_electricity
households_space_heater_heatpump_air_water_electricity
households_space_heater_heatpump_ground_water_electricity
households_space_heater_hybrid_heatpump_air_water_electricity
households_space_heater_hybrid_heatpump_air_water_electricity_electricity
households_space_heater_hybrid_heatpump_air_water_electricity_gas
households_space_heater_micro_chp_network_gas
households_space_heater_network_gas

buffer_water_heating
households_water_heater_combined_network_gas
households_water_heater_district_heating_steam_hot_water
households_water_heater_fuel_cell_chp_network_gas
households_water_heater_heatpump_air_water_electricity
households_water_heater_heatpump_ground_water_electricity
households_water_heater_hybrid_heatpump_air_water_electricity
households_water_heater_hybrid_heatpump_air_water_electricity_electricity
households_water_heater_hybrid_heatpump_air_water_electricity_gas
households_water_heater_micro_chp_network_gas
households_water_heater_network_gas
households_water_heater_resistive_electricity
````

**Comment by ChaelKruip** [comment]
> Is this for every technology that has a distinction between hot water and space heating?

Yes. 

> Or are we just talking buffers?

Every household has two buffers now. But connected to these buffers are technologies. I don't understand the question or the confusion 😕

**Comment by dennisschoenmakers** [comment]
> To really solve this in an elegant way, we need to merge space heating and hot water combi-options in the ETM first.

I strongly agree. We should try to fix it in the ETM to start with.

**Comment by jorisberkhout** [comment]
> > To really solve this in an elegant way, we need to merge space heating and hot water combi-
> options in the ETM first.
>
> I strongly agree. We should try to fix it in the ETM to start with.

I agree with both of you, @ChaelKruip and @dennisschoenmakers . @ChaelKruip and I wrote a proposal for this merge that we never implemented because of time constraints. The proposal seems very valid still. More information on this proposal can be found in https://github.com/quintel/etmodel/issues/1595.

**Comment by jorisberkhout** [comment]
Related to https://github.com/quintel/etmodel/issues/2185
